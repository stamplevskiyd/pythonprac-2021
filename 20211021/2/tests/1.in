:cos x sin(x)
cos -3
:decorate s "--<<{}>>--".format(s)
decorate "ЖЖЖ"
cos 2
quit 'все'
