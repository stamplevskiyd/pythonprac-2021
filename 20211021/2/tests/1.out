-0.1411200080598672
--<<ЖЖЖ>>--
0.9092974268256817
3:6
